servers/graphql-kotlin-ktor-server/src/test/kotlin/com/expediagroup/graphql/server/ktor/GraphQLPluginTest.kt

```diff
@@ -180,10 +182,10 @@ class GraphQLPluginTest {
     }
 
     @Test
-    fun `server should return Bad Request for invalid POST requests`() {
+    fun `server should return Unsupported Media Type for invalid POST requests`() {
```

Comment:
I think this can be "fixed" if we set the content type to json in the request. 415 is due to invalid content type or encoding. See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415

I think the server is correctly responding 415 to the test case in this scenario based on default ktor handling

This can be seen in their tests here: https://github.com/ktorio/ktor/blob/7c76fa7c0f2b7dcc6e0445da8612d75bb5d11609/ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/routing/RoutingProcessingTest.kt#L535-L595